openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -681,19 +681,21 @@ const EntityTable = ({
         dataSource={data}
         expandable={{
           defaultExpandedRowKeys: [],
-          expandIcon: ({ expanded, onExpand, record }) =>
-            record.children ? (
-              <FontAwesomeIcon
-                className="tw-mr-2 tw-cursor-pointer"
-                icon={expanded ? faCaretDown : faCaretRight}
+          rowExpandable: () => true,
+          expandIcon: ({ expanded, onExpand, record }) => {
+            return record.children ? (
```

Comment:
Expand icon should just return icon based on expand state and not decided which row should be expanded